contributing/naming-conventions.md

```diff
@@ -74,7 +74,8 @@ Props for event handlers should include an `on` prefix:
 +onHover
 ```
 
-Boolean props should have an auxiliary verb prefix:
+Boolean props should have an auxiliary verb prefix _unless_ the boolean prop maps directly
+to a boolean html attribute:
```

Comment:
@rey-wright how do you feel about this?

Reply:
@akdetrick yep this is it. We sort of already know we do this (for instance, we don't make `isDisabled` props, especially if the element already has `disabled` natively, as it's yet another thing that the user has to "learn". I sort of like the verb to mostly communicate that it's a boolean, "because" you aren't already familiar with this prop. We don't have that issue with existing boolean props that map onto the underlying html element because the developer already knows those.